Makefile

```diff
@@ -7,8 +7,6 @@ APP_VERSION_FILE = app/version.py
 GIT_BRANCH ?= $(shell git symbolic-ref --short HEAD 2> /dev/null || echo "detached")
 GIT_COMMIT ?= $(shell git rev-parse HEAD)
 
-CF_API ?= api.cloud.service.gov.uk
-CF_ORG ?= govuk-notify
 CF_SPACE ?= ${DEPLOY_ENV}
```

Comment:
[question] do we want to remove all cloud-foundry functions from this file?

Reply:
The remaining ones are basically the "failwhale" infrastructure, which I don't think we've discussed enough to remove. I imagine that'll migrate somewhere else, but in the meantime I figured we could keep these commands handy.